playground/README.md

```diff
@@ -0,0 +1,187 @@
+  files by path and/or use client-side routing
+- JavaScript and TypeScript files can be used interchangeably, with
+  TypeScript files being transpiled on the fly; when referring to
+  script files, always use the `.js` extension, regardless of
+  whether the source file is JavaScript or TypeScript
+- Other subresources (CSS, images, fonts, etc.) may be loaded as
+  usual, relative to the server root `/` or the referring file
+  `./`
+- Usage of Vite-specific features (loaders, etc.) is discouraged
+
+### Dependencies
+
+- If an example depends only on Lit monorepo packages, there is
+  no need to include a `package.json` (though the template example
+  does include an empty `package.json` for convenience)
+- If an example depends on external packages or requires a specific
+  released version of a Lit monorepo package, just add a
+  `package.json` file and use the `npm` CLI directly in the example
+  folder to manage and install dependencies
+
+### Lit.dev playground compatibility
+
+- Since Gists don't support nested folders, you should use a flat
+  structure for any example you might want to export
+- The Lit.dev playground expects a single HTML entrypoint, at
+  `index.html`
+- The Lit.dev playground supports implicit (undeclared)
+  dependencies, but the monorepo playground does not (with the
+  exception of Lit monorepo packages)
+- The provided `import` script constructs a `package.json`, derived
+  from the Gist's original `package.json` (if any) and source files
+  - If the original `package.json` explicitly depends on any Lit
+    monorepo packages, these dependencies are omitted from the
+    constructed `package.json` so that they will be loaded from the
+    current monorepo source instead
+  - If the Gist's JavaScript or TypeScript source files import any
+    modules from external packages that are not included in the
+    Gist's `package.json`, these packages are added to the
+    constructed `package.json` (though only top-level, static
+    imports are detected)
+- Upon successful completion, the `import` script writes a file
+  named `_import.info.json` to the example folder, containing the
+  source Gist ID and the contents of the Gist's original
+  `package.json` file
+- The `export` script requires manual installation of
+  [this gist CLI](https://github.com/defunkt/gist)
+- The `import` and `export` scripts do minimal error detection;
+  they're designed to "just work" for common cases but don't
+  enforce compliance with structural requirements and may be
+  tripped up by edge cases
+
+  ### Implementation details
+
+- The `@lit-internal/playground` package is a workspace within
+  the monorepo so that its dependencies (TypeScript and Vite)
+  are installed via `npm install` at the monorepo root
+- Individual examples are not workspaces; each example's
+  external dependencies (if any) are installed locally
+- Examples use the [Vite](https://vitejs.dev) dev server
```

Comment:
The default tsconfig/vite setup doesn't support using the `accessor` keyword. Given our need to support this for standard decorators and its superior integration with class fields, it'd be great to make this work.

I tried some different settings but wasn't able to make it work, either via configuring ts or esbuild.

Reply:
Thanks for reporting—I'll take a look and see if I can make it work!

Reply:
I just pushed an update that allows standard decorators to be used on a per-example basis, and added an example to illustrate. It requires inclusion of a `tsconfig.json` in the example folder, but that can be a one-liner that just imports a standard-decorator-friendly base config, which I've added to the `playground` folder.

If this is too much of a pain, we might consider adding an option to the `npm run new` script that would add the `tsconfig.json` upon creation.